src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/LocationMetaRepository.cs

```diff
@@ -153,16 +146,58 @@ FROM read_csv('{dataSetVersionPathResolver.CsvDataPath(dataSetVersion):raw}', AL
         }
     }
 
+    private async Task<List<LocationOptionMetaRow>> GetLocationOptionMetas(
+        IDuckDbConnection duckDbConnection,
+        DataSetVersion dataSetVersion,
+        CancellationToken cancellationToken,
```

Comment:
Lint here - the cancellation token should be the last parameter

Reply:
Cool, amended thanks